### PR TITLE
General: Add Storybook tests to CI

### PR DESCRIPTION
**Pull request by micieslak**
### What does the PR do

Running on CI all tests from `Storybook` directory via `ctest`:
- tests for Storybook itself
- UI unit tests using the same stubs mechanism as Storybook pages
- PagesValidator checking if SB pages are compilable

Closes: #12213
Closes: #12437

### Affected areas
`ci` directory

### CODE REVIEW

File: ci/Jenkinsfile.tests-ui [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+      artifactNumToKeepStr: '3',
+    ))
+    /* Abort old PR builds. */
+    disableConcurrentBuilds(
+      abortPrevious: isPRBuild
+    )
+  }
+
+  environment {
+    TARGET = 'tests/statusq'
```
Comment (jakubgs) [inline ci/Jenkinsfile.tests-ui:37]: You didn't update the `TARGET`.

Comment (micieslak) [reply ci/Jenkinsfile.tests-ui]: Fixed, thanks.

**Review by caybro — APPROVED** [review]
LGTM

**Review by alexjba — APPROVED** [review]
🏆

**Comment by alexjba** [comment]
@micieslak This workarounds sounds good to me!

I'd create a task to remove it and add this https://github.com/status-im/status-desktop/issues/9350 as dependency. 
Things are now much more simple because we're free to drop the support for Ubuntu 16.04 and use 20.04 which is compatible with the prebuilt Qt binaries.

**Comment by micieslak** [comment]
> @micieslak This workarounds sounds good to me!
> 
> I'd create a task to remove it and add this #9350 as dependency. Things are now much more simple because we're free to drop the support for Ubuntu 16.04 and use 20.04 which is compatible with the prebuilt Qt binaries.

Good idea! Done: https://github.com/status-im/status-desktop/issues/12474

**Pull request merged**